src/source/TMC5130Stepper.cpp

```diff
@@ -14,14 +14,14 @@ void TMC5130StepperBase<T>::begin() {
 
   if (this->uses_sw_spi) TMC_SW_SPI.init();
 
-  GCONF(this->GCONF_register.sr);
-  CHOPCONF(this->CHOPCONF_register.sr);
-  COOLCONF(this->COOLCONF_register.sr);
-  PWMCONF(this->PWMCONF_register.sr);
-  IHOLD_IRUN(this->IHOLD_IRUN_register.sr);
+  this->GCONF(this->GCONF_register.sr);
+  this->CHOPCONF(this->CHOPCONF_register.sr);
+  this->COOLCONF(this->COOLCONF_register.sr);
+  this->PWMCONF(this->PWMCONF_register.sr);
+  this->IHOLD_IRUN(this->IHOLD_IRUN_register.sr);
```

Comment:
@ejtagle — Do you think these should be `SELF.` also? In case this class becomes a base class for something with its own implementations in future?

Reply:
SELF instead of this-> does not hurt, but it is only required if the method to call is "virtual" ...